xml/depl_maintenance.xml

```diff
@@ -508,6 +508,258 @@
+    </para>
+    <!-- FIXME: Currently doesn't work according to devs
+	 <para>
+     If &rabbit; is using DRBD, enable the <guimenu>Use RabbitMQ
+     Clustering</guimenu> option in the &rabbit; &barcl; under <guimenu>High Availability</guimenu>. Since DRBD is not available
+     in &productname; 8, and &rabbit; clustering is supported in &productname;
+     7, you should switch to this option now.
+    </para> -->
+    <para>
+     Follow <xref linkend="postgre.mariadb.data.migration"/> to migrate the data from &postgres; to &mariadb;.
+    </para>
+    <procedure xml:id="postgre.mariadb.data.migration">
+     <title>Data Migration</title>
+     <step>
+      <para>
+           Run the <systemitem>/opt/dell/bin/prepare-mariadb</systemitem>
+	   script on the &admnode; to prepare the &mariadb; instance by creating
+	   the required users, databases, and tables.
+      </para>
+     </step>
+     <step>
+      <para>
+       After the script is finished, you'll find a list of all databases and
+       URLs that are ready for data migration in the
+       <filename>/etc/pg2mysql/databases.yaml</filename> located on one of the &dbnode;s. The script's output
+      may look as follows:</para>
+      <screen>Preparing node d52-54-77-77-01-01.vo6.cloud.suse.de
+Adding recipe[database::pg2mariadb_preparation] to run_list
+Running chef-client on d52-54-77-77-01-01.vo6.cloud.suse.de...
+Log: /var/log/crowbar/db-prepare.chef-client.log on
+d52-54-77-77-01-01.vo6.cloud.suse.de Run time: 444.725193199s
+Removing recipe[database::pg2mariadb_preparation] from run_list
+Prepare completed for d52-54-77-77-01-01.vo6.cloud.suse.de
+Summary of used databases: /etc/pg2mysql/databases.yaml on
+d52-54-77-77-01-01.vo6.cloud.suse.de</screen>
+      <para>The <literal>Summary of used databases:</literal> line shows the
+      exact location of the <filename>/etc/pg2mysql/databases.yaml</filename> file.
+      </para>
+      <para>
+       The <filename>/etc/pg2mysql/databases.yaml</filename> file contains a
+       list of databases along with their source and target connection strings:
+      </para>
+<screen>keystone:
+  source: postgresql://keystone:vZn3nfxXzv97@192.168.243.87/keystone
+  target: mysql+pymysql://keystone:vZn3nfxXzv97@192.168.243.88/keystone?charset=utf8
+glance:
+  source: postgresql://glance:cOau7NhaA54N@192.168.243.87/glance
+  target: mysql+pymysql://glance:cOau7NhaA54N@192.168.243.88/glance?charset=utf8
+cinder:
+  source: postgresql://cinder:idRll2gJPodv@192.168.243.87/cinder
+  target: mysql+pymysql://cinder:idRll2gJPodv@192.168.243.88/cinder?charset=utf8</screen>
+     </step>
+     <step>
+      <para>
+       Install the <package>python-psql2mysql</package> package on the
+       &admnode; or directly on the &dbnode;.
+      </para>
+      <para>
+       Running the tool directly on &dbnode; speeds up the migration operation,
+       as this does not require moving database data over the network. You can
```

Comment:
Hm, this is probably a bit too simplified. The migration will still require network traffic (at the bare mininum for the galera replication). However running it on a controller node might reduce the traffice if that e.g. is the currently (active) postgresql node. Ideally we would make sure to have the active postgresql, haproxy and mariadb node on the same machine. But there is currently nothing in the tools that really ensure this.

Reply:
So maybe we should just tell them to run it on database node and do not go into details?

Reply:
Fixed.